tests/test_date.py

```diff
@@ -2,66 +2,135 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+from types import MethodType
+from itertools import izip
+from collections import OrderedDict
 import unittest
 from datetime import datetime, timedelta
 
 from dateutil.relativedelta import relativedelta
+
```

Comment:
Outside of the blank line and reviewing the docs for import order, I do not see what is out of order here?